gdal/ogr/ogrsf_frmts/mitab/mitab_tabfile.cpp

```diff
@@ -978,6 +997,22 @@ int TABFile::WriteTABFile()
         {
             VSIFPrintfL(fp, "Definition Table\n");
             VSIFPrintfL(fp, "  Type NATIVE Charset \"%s\"\n", m_pszCharset);
+            const char *pszDescription = GetMetadataItem(DESCRIPTION_KEY);
+            if (nullptr != pszDescription)
+            {
+                const char *pszEncoding = GetEncoding();
+                if (nullptr == pszEncoding || EQUAL(pszEncoding, ""))
```

Comment:
what about if the description contains a double-quote character ?

Reply:
Fixed. Also added check for line breaks.